lib/jsonapi/error_view.ex

```diff
@@ -93,12 +94,57 @@ defmodule JSONAPI.ErrorView do
     |> serialize_error
   end
 
+  @spec relationships_missing_object :: map()
+  def relationships_missing_object do
+    "Relationships parameter is not an object"
+    |> build_error(
+      400,
+      "Check out https://jsonapi.org/format/#document-resource-object-relationships for more info.",
+      "/data/relationships"
+    )
+    |> serialize_error
+  end
+
+  @spec missing_relationship_data_param_error(binary()) :: map()
+  def missing_relationship_data_param_error(relationship_name) do
+    "Missing data member in relationship"
+    |> build_error(
+      400,
+      "Check out https://jsonapi.org/format/#crud-creating and https://jsonapi.org/format/#crud-updating-resource-relationships for more info.",
+      "/data/relationships/#{relationship_name}/data"
+    )
+  end
+
+  @spec missing_relationship_data_id_param_error(binary()) :: map()
+  def missing_relationship_data_id_param_error(relationship_name) do
+    "Missing id in relationship data parameter"
+    |> build_error(
+      400,
+      @relationship_resource_linkage_message,
+      "/data/relationships/#{relationship_name}/data/id"
+    )
+  end
+
+  @spec missing_relationship_data_type_param_error(binary()) :: map()
+  def missing_relationship_data_type_param_error(relationship_name) do
+    "Missing type in relationship data parameter"
+    |> build_error(
+      400,
+      @relationship_resource_linkage_message,
+      "/data/relationships/#{relationship_name}/data/type"
+    )
+  end
+
   @spec send_error(Plug.Conn.t(), term()) :: term()
   def send_error(conn, %{errors: [%{status: status}]} = error),
     do: send_error(conn, status, error)
 
   def send_error(conn, %{errors: errors} = error) when is_list(errors) do
-    status = Enum.max_by(errors, &Map.get(&1, :status))
```

Comment:
Best I can tell this was not called on the latest stable version prior to this PR. I think this is a safe change as long as it works in the new workflows.